wavelength-plugins/icons.js

```diff
@@ -1,84 +1,85 @@
-/*
-* icons.js
-* Credits: Lord Haji, panpawn
-*/
-'use strict';
+/******************************
+* Icons for Pokemon Showdown  *
+* Credits: Lord Haji, panpawn.*
+*******************************/
+"use strict";
 
+let iconsData = FS("config/icons.json").readIfExistsSync();
 let icons = {};
-const fs = require('fs');
 
-function load() {
-	fs.readFile('config/icons.json', 'utf8', function (err, file) {
-		if (err) return;
-		icons = JSON.parse(file);
-	});
+if (iconsData) {
+   icons = JSON.parse(icons);
 }
-load();
 
 function updateIcons() {
-	fs.writeFileSync('config/icons.json', JSON.stringify(icons));
+	FS("config/icons.json").writeUpdate(() => (
+		JSON.stringify(icons)
+	));
 
-	let newCss = '/* ICONS START */\n';
+	let newCss = "/* ICONS START */\n";
 
 	for (let name in icons) {
 		newCss += generateCSS(name, icons[name]);
 	}
-	newCss += '/* ICONS END */\n';
+	newCss += "/* ICONS END */\n";
 
-	let file = fs.readFileSync('config/custom.css', 'utf8').split('\n');
-	if (~file.indexOf('/* ICONS START */')) file.splice(file.indexOf('/* ICONS START */'), (file.indexOf('/* ICONS END */') - file.indexOf('/* ICONS START */')) + 1);
-	fs.writeFileSync('config/custom.css', file.join('\n') + newCss);
+	let file = FS("config/custom.css").readIfExistsSync().split("\n");
+	if (!file.includes("/* ICONS START */")) file.splice(file.indexOf("/* ICONS START */"), (file.indexOf("/* ICONS END */") - file.indexOf("/* ICONS START */")) + 1);
```

Comment:
I'm sorry; remove the `!`. I was being dumb.

Reply:
ok